### PR TITLE
Add support for installing default global npm packages on nvm install

### LINKED ISSUE #178: Add support for nvm style default packages
`nvm` has support for [Default Packages](https://github.com/nvm-sh/nvm#default-global-packages-from-file-while-installing), where one can define a set of global packages that are installed during each install of a `node` version.

I'd love to see this introduced into nvm.fish. I frequently need to have `yarn`, for example, installed into my fresh node environments.

In fact, I already implemented this for myself. A PR is soon to follow for your consideration.

### PR DESCRIPTION
**Pull request by kallja**
closes #178

### CODE REVIEW

**Comment by smithshelke** [comment]
Until this gets merged, how are you able work around installing global npm packages

**Comment by jorgebucaran** [comment]
Sorry, I usually never install global packages, but I'd like to see this merged still. It's just that I need a bit of time to sit down and understand what this does and make sure it does it in a way that makes sense. 🙏

File: functions/nvm.fish [hunk 1]
```diff
@@ -97,6 +97,10 @@ function nvm --argument-names cmd v --description "Node version manager"
 
                 echo -en "\033[F\33[2K\x1b[0m"
 
+                if not set -q $nvm_default_packages[1]
+                    set install_default_packages true
+                end
+
```
Comment (jorgebucaran) [inline functions/nvm.fish]: @kallja What's the purpose of this code?

File: functions/nvm.fish [hunk 2]
```diff
@@ -108,6 +112,11 @@ function nvm --argument-names cmd v --description "Node version manager"
             if test $v != "$nvm_current_version"
                 set --query nvm_current_version && _nvm_version_deactivate $nvm_current_version
                 _nvm_version_activate $v
+
+                if test $install_default_packages
+                    echo "Installing default npm packages: $nvm_default_packages"
+                    npm install -g $nvm_default_packages
+                end
             end
```
Comment (jorgebucaran) [inline functions/nvm.fish]: Think we could boil this down to:

```fish
if set --query nvm_default_packages[1]
    npm install --global $nvm_default_packages
end
```

**Review by jorgebucaran — CHANGES_REQUESTED** [review]

**Pull request merged**